src/dropin-model.js

```diff
@@ -205,13 +206,27 @@ function getSupportedPaymentOptions(options) {
 
 function isPaymentOptionEnabled(paymentOption, options) {
   var gatewayConfiguration = options.client.getConfiguration().gatewayConfiguration;
+  var applePayEnabled, applePayBrowserSupported;
 
   if (paymentOption === 'card') {
     return gatewayConfiguration.creditCards.supportedCardTypes.length > 0;
   } else if (paymentOption === 'paypal') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypal);
   } else if (paymentOption === 'paypalCredit') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypalCredit);
+  } else if (paymentOption === 'applePay') {
+    applePayEnabled = gatewayConfiguration.applePay && Boolean(options.merchantConfiguration.applePay);
+    applePayBrowserSupported = window.ApplePaySession && ApplePaySession.canMakePayments(); // eslint-disable-line no-undef
+
+    if (!applePayEnabled) {
+      return false;
+    }
+    if (!applePayBrowserSupported) {
+      console.log('Browser does not support Apple Pay.'); // eslint-disable-line no-console
```

Comment:
Do we want to console log this or just document that Apple Pay will render in supported browsers in the docs?